Drop Database code from libs/model (fixes #26)

addyosmani opened the pull request:
r: @paullewis 

Tested locally from a fresh cache and everything appears to work. Requesting a sanity check in case I've accidentally broken something model related in the process.

addyosmani committed:
Drop Database code from libs/model (fixes #26)

paullewis commented on src/scripts/model/Model.js:
```diff
@@ -39,160 +36,8 @@ export default class Model {
-        });
-  }
 
   put() {
```
You have put but no get?

addyosmani replied:
Oof. Noobish error. I'll get that fixed up.

addyosmani commented:
Fixed this up locally and added a few other methods that were missing, but as it turns out we don't actually use Models at all in app-shell. Matt's going to refactor them out. Closing this PR.
